packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -516,6 +516,7 @@ function FontLibraryProvider( { children } ) {
 				libraryFontSelected,
 				handleSetLibraryFontSelected,
 				fontFamilies,
+				setFontFamilies,
```

Comment:
Let's avoid adding a new context. We can simply add the following directly to the `installed-fonts.js` file:

```javascript
const [ fontFamilies, setFontFamilies ] = useGlobalSetting(
	'typography.fontFamilies'
);
```